web/src/modules/feedback/components/FeedbackBoxContainer.js

```diff
@@ -237,9 +240,10 @@ export class FeedbackBoxContainer extends React.Component<PropsType, StateType>
 
   handleSubmit = async () => {
     const { onSubmit } = this.props
-    const { selectedFeedbackOption, comment } = this.state
+    const { selectedFeedbackOption, comment, contactMail } = this.state
     try {
-      await this.postFeedbackData(this.getFeedbackData(selectedFeedbackOption, comment))
+      await this.postFeedbackData(
+        this.getFeedbackData(selectedFeedbackOption, `${comment}    contactMail: ${contactMail}`))
```

Comment:
```suggestion
        this.getFeedbackData(selectedFeedbackOption, `${comment}\m\nKontakt Adresse: ${contactMail}`))
```

I suggest to make this string german as all people who read comments are German

Reply:
Yes, you're right